module/actor/sheets/WitcherActorSheet.js

```diff
@@ -337,6 +337,10 @@ export default class WitcherActorSheet extends ActorSheet {
                         let newCritList = [];
                         critList.forEach(crit => {
                             crit.daysHealed += 1;
+                            // Reduce the healing time by 2 days if sterFluid was used
+                            if (sterFluid) {
```

Comment:
This is already factored into the healing time line 309

Reply:
The code is specifically designed to decrease the maximum recovery time for a critical injury by 2 days whenever a specific item is used

Reply:
The code at line 309 adds 2 days when it is used. So this does the same

Reply:
ah, ok. I will remove this code

Reply:
I also already commited your fix for the path building in another feature I added today

Reply:
ok

> I also already commited your fix for the path building in another feature I added today

Reply:
I will remove these two commits